spec/unit/ipaddress6_spec.rb

```diff
@@ -44,8 +50,11 @@ def ifconfig_fixture(filename)
   it "should return ipaddress6 with fe80 in any other octet than the first for Linux" do
     Facter::Core::Execution.stubs(:exec).with('uname -s').returns('Linux')
     Facter::Util::IP.stubs(:get_ifconfig).returns("/sbin/ifconfig")
-    Facter::Util::IP.stubs(:exec_ifconfig).with(["2>/dev/null"]).
+    Facter::Util::IP.stubs(:exec_ifconfig).
       returns(ifconfig_fixture('linux_ifconfig_all_with_multiple_interfaces_and_fe80'))
+    routes = ifconfig_fixture('net_route_all_with_multiple_interfaces')
+    Facter::Util::IP.stubs(:read_proc_net_route).returns(routes)
+    Facter.collection.internal_loader.load(:interfaces)
 
     Facter.value(:ipaddress6).should == "2610:10:20:209:212:3fff:fe80:2201"
   end
```

Comment:
It looks like the next test needs to be updated as well, it's still stubbing ifconfig with "2>/dev/null" and produces an error message on Windows: "Could not retrieve fact='ipaddress6', resolution='<anonymous>': No such file or directory @ rb_sysopen - /proc/net/route"